torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -327,6 +330,10 @@ def powerSGD_hook(
         bucket (dist.GradBucket): Bucket that stores a 1D flattened gradient tensor that batches multiple per-variable tensors.
             Note that since DDP comm hook only supports single process single device mode,
             only exactly one tensor is stored in this bucket.
+        batch_tensors_with_same_shape (bool): A boolean value that when set to ``True``, compress and decompress tensors with
```

Comment:
This needs to be placed inside `PowerSGDState`. A valid hook cannot accept a third arg.

Reply:
Nit: Replace "A boolean value that when set to ``True``," by "Whether to".